### ISSUE TITLE
About the training set Di

### ISSUE THREAD

**Issue by JackySnake** [ZihanWangKi/CrossWeigh#2]
Hello, I am reading this paper and very intrested in it. There is a point that I can not understand.
In Eq.2, the i-th test_entities get all entities in the Di. Then, Eq.3 remove all the sentences which labeled entities contained in the test_entities.
I think there only the sentences remained which labeled only 'O', if so, the model should have no ability to recognize the type entities. How could the model to disticnt the label mistake?
I am looking forward for your reply, thanks a lot.

**Comment by ZihanWangKi** [comment]
Hi Jacky,
In Eq.2, the ej means the named entities (or surface names). For example, if _I went to [Chicago]{LOC}_ is in the test fold, then the sentence _I went to [New York]{LOC}_ in the training fold will **NOT** be removed. However, the sentence _I played football in [Chicago]{LOC}_ will be removed since it contains the same entity.
Hope this answers your question.

**Comment by JackySnake** [comment]
So,the test_entities only contain the entities in the test set of the Di,is that right?
Is the test set in each fold sepreated from the train set or test set?

**Comment by ZihanWangKi** [comment]
Yes, for each fold, the test_entities contains only the entities in that fold.
You can think the whole process as cross-validation. First, we separate the train set into k folds, we each fold, we treat it as a _test fold_ and the other k - 1 folds as a _training set_. Then, we remove all the sentences in the _training set_ that have an entity appearing in the _test fold_.

**Comment by JackySnake** [comment]
Thank you for your reply, I understand.
